pkg/features/kube_features.go

```diff
@@ -217,22 +169,6 @@ const (
 	// NodePublishVolume calls.
 	DelegateFSGroupToCSIDriver featuregate.Feature = "DelegateFSGroupToCSIDriver"
 
-	// owner: @jiayingz, @swatisehgal (for GA graduation)
-	// alpha: v1.8
-	// beta: v1.10
-	// GA: v1.26
-	//
-	// Enables support for Device Plugins
-	DevicePlugins featuregate.Feature = "DevicePlugins"
```

Comment:
this says `GA: v1.26`. Typically we wait for two releases to remove the feature gate

Reply:
Thanks for your point @SergeyKanzhelev, Actually, this feature gate doesn't have any code references, so I think it's reasonable to remove it.

Reply:
this is by design. Feature Gate stays in the list of feature gates for two releases so environments where it is configured will not break. See https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-a-feature-or-behavior

/lgtm cancel

Reply:
Thanks! Now updated.